gnome-initial-setup/pages/keyboard/cc-input-chooser.c

```diff
@@ -169,7 +169,14 @@ preview_cb (GtkLabel       *label,
 	InputWidget *widget;
 	const gchar *layout;
 	const gchar *variant;
-	gchar *commandline;
+	gchar *lc_msg = setlocale (LC_MESSAGES, NULL);
+	gchar *command[] = {"/usr/bin/gkbd-keyboard-display", "-l", NULL, NULL};
+	gchar **env;
+	GError *error = NULL;
+	GPid pid;
+
+	env = g_get_environ ();
+	env = g_environ_setenv (env, "LC_MESSAGES", lc_msg, TRUE);
```

Comment:
It is the same thing because the language page update the locale (https://github.com/endlessm/gnome-initial-setup/pull/172/files#diff-0f1f52d636a1c9f7f830f8b33b5991acR129) and the keyboard page will read the new value.
@cosimoc